specs/consensus.md

```diff
@@ -101,6 +102,8 @@ The validity of a newly-seen block, `block`, is determined by two components, de
 1. [Block structure](#block-structure): whether the block header is valid, and data in a block is arranged into a valid and matching data root (i.e. syntax).
 1. [State transition](#state-transitions): whether the application of transactions in the block produces a matching and valid state root (i.e. semantics).
 
+Pseudocode in this section is not in any specific language and should be interpreted as being in a neutral and sane language.
```

Comment:
Is it at least consistent across the spec?

Reply:
Yes, it should all be consistent.

Reply:
It would probably be better to use Python syntax across the spec, since it's almost identical but it's a real language so there's a point of reference

Reply:
It might indeed be a good idea to make the code parts actually executable, and probably in python. This would also tie in nicely to building a set of test vectors for the spec. But we can leave that for a future issue.